source/_posts/2017/2017-08-14-how-to-apply-nette-coding-standard-in-your-project.md

```diff
@@ -0,0 +1,109 @@
+---
+id: 50
+layout: post
+title: "How to Apply Nette Coding Standard in Your Project"
+perex: '''
+    Coding standards help you to <strong>focus on architecture issues rather than code details in pull-requests</strong>, moreover in open-source. <strong>I wanted to make this possible in Nette</strong>, but Coding Standards could be found only in <a href="https://nette.org/en/coding-standard">documentation</a>.
```

Comment:
Well, I'll need to change this, because my goal is quite different.

Thanks to automated coding standard pull-requests are more fun. You don't have expalin where to put space or bracket, but you can talk about architecture or meaning of the code.